### ISSUE TITLE
Path based sidebar

### ISSUE THREAD

**Issue by stereobooster** [enzonotario/vitepress-openapi#119]
### What would you like?

I wonder if it is possible to implement path based sidebar. For example, we have

```
/api/v1/products CRUD
/api/v1/orders CRUD
```

Sidebar would look like

```
/api/v1
  ├ products
  |  ├ PUT
  |  ├ GET
  |  ├ POST
  |  └ DELETE
  └ orders
     ├ PUT
     ├ GET
     ├ POST
     └ DELETE
```

**Note**: empty folders are collapsed

### Why is this needed?

I want to use more than one tag per endpoint. But in this case sidebar looks strange. Maybe there is a way to organize sidebar better even if there are multiple tags...

**Comment by enzonotario** [comment]
Hi @stereobooster, I'm working on it in #132. 

Can you please validate it via [pkg-pr-new release](https://github.com/enzonotario/vitepress-openapi/pull/132#issuecomment-2533235568)? You can also check this [StackBlitz](https://stackblitz.com/edit/enzonotario-vitepress-openapi-starter-y7glvteq?file=docs%2F.vitepress%2Fconfig.ts).

There are a few [options to customize it](https://vitepress-openapi-git-feature-sid-cecb10-enzo-notarios-projects.vercel.app/layouts/sidebar.html#items-by-paths).

Any feedback is welcome, thanks!

**Comment by enzonotario** [comment]
Nice, it's now released in [v0.0.3-alpha.52](https://github.com/enzonotario/vitepress-openapi/releases/tag/v0.0.3-alpha.52).

Thanks!
